todo_MVC1/category.rb

```diff
@@ -0,0 +1,24 @@
+require 'active_record'
+
+class Category < ActiveRecord::Base
+	establish_connection adapter: 'sqlite3',
+						 database: 'todomvc.db'
+	
+	validates_presence_of :category					 
```

Comment:
I'd avoid having an attribute name that's the same as the class name - when you start saying things like "I have a variable category of class Category with a category" you sound like either Xzibit or a smurf.